master/getting-started/kubernetes/upgrade/index.md

```diff
@@ -4,23 +4,36 @@ redirect_from: latest/getting-started/kubernetes/upgrade
 no_canonical: true
 ---
 
+Upgrading to this release of Calico from a pre-v3.0 release may require some manual steps, depending on
+your choice of datastore.
 
-## Prerequisites
+## Before you begin
 
 - You must [upgrade](/v2.6/getting-started/kubernetes/upgrade) 
-  to a tagged release of {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) before you can upgrade to {{site.prodname}} {{site.data.versions[page.version].first.title}}. 
-
-- An [etcdv3 cluster](https://coreos.com/etcd/docs/latest/). 
-
+  to a release of {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) 
+  (or a later v2.6.x release) before you can upgrade to {{site.prodname}} 
+  {{site.data.versions[page.version].first.title}}. 
+  
+  > **Important**: Calico v2.6.4 was a special transitional release that included changes to enable 
```

Comment:
Should all occurrences of 'Calico' be {{site.prodname}} ?
(If so, does that actually work, bearing in mind different versions?)